### ISSUE TITLE
Call for maintainers 📞

### ISSUE THREAD

**Issue by addyosmani** [tastejs/todomvc#1842]
**tl;dr: interested in helping maintain TodoMVC? Comment here and let's chat.** 

When @tastejs/todomvc first started, we were fortunate enough to have a new "lead" maintainer more or less each year. In the last few years, with the core team shifting their focus to other projects, we've had less time to review PRs, chat about project direction and approve or reject new app suggestions. 

This hasn't stopped the community from continuing to build TodoMVC implementations. Today alone I've seen new apps using [WASM](https://twitter.com/rauchg/status/945442304940064768), VueJS, Rust, Elm. The list goes on and it would be great for the project to stay up to date with implems being submitted and link out to those that aren't where they can provide value to those wanting to learn.

There's also work to be done coalescing more up to date TodoMVC implementations that happen to be hosted elsewhere. For example, I and @mathiasbynens help maintain the [Speedometer](https://webkit.org/blog/3395/speedometer-benchmark-for-web-app-responsiveness/) benchmark where we've recently updated most TodoMVC apps here to their latest versions, but host them in the benchmark repo (WebKit trunk) with some minor differences (e.g persistence is disabled).

If folks are interested in getting involved in the project, we would ❤️ some help with the following:

* Code reviewing PRs
* Closing PRs that are duplicates of newer/older better ones
* Reviewing issues (e.g are there bugs that impact all apps?)
* Helping with project direction
* Helping with project advocacy (e.g helping maintain the project Twitter account)

If you've got time to help out, comment and let us know what you'd like to help with. Sure we can help answer any questions and assist with on-boarding.

**Comment by howardroark** [comment]
I'd be glad to help sort/close PRs :)

**Comment by FadySamirSadek** [comment]
I would be more than happy to help with code reviewing PRs , reviewing issues and closing PRs that are duplicates .

**Comment by bernard-lin** [comment]
Happy to help with maintenance here too!

**Comment by nithindavid** [comment]
Hey, I could help with 
1. Code reviewing PRs
2. Closing PRs that are duplicates of newer/older better ones 

😄

**Comment by addyosmani** [comment]
Hey folks. Thanks @bernard-lin @nithindavid @FadySamirSadek @howardroark for your interest. We really appreciate it. 

Let's try to get the project back on track! There are currently 142 other issues open and 61 open pull requests. Working on the project tends to involve reviewing three types of changes:

1) Fixes to existing implementations
2) Fixes to continuous integration / tests
3) PRs to add in new implementations

It would help a lot if we had some folks focused on looking at 1) and 2) and others looking at 3). 

We're usually conservative about bringing in new implementations just because there's a maintenance cost that builds up over time 🙃 So a "React TodoMVC" or "React + Redux TodoMVC" was fine, but newer libraries/frameworks that have very low traction tend to be entries we'll link to from our wiki until they've got enough momentum to justify maintaining the apps ourselves here. 

If you've got the time, it would be great to start helping out with reviews and we'll start adding commit/contributor rights to your accounts as we see you get involved.

Going to fyi @samccone in case he starts seeing a few more folks helping with reviews :)

**Comment by howardroark** [comment]
I know one common pattern is that PRs tend to lack updates that keep the tests passing.  Then they sit untouched for a long time.

Perhaps it would be helpful to setup a of bot which pokes all PRs where tests fail and there is no recent activity, then closes them if tests don't start to pass soon after.

**Comment by bernard-lin** [comment]
Sounds great! Thanks @addyosmani

**Comment by addyosmani** [comment]
Cheers! 

@howardroark 

> Perhaps it would be helpful to setup a of bot which pokes all PRs where tests fail and there is no recent activity, then closes them if tests don't start to pass soon after.

I'd be open to us experimenting with a bot like this. One challenge we regularly have is tests of the main repo being broken (the build currently is) which might be something we'd need to fix before introducing a bot.

**Comment by whitefire0** [comment]
I'm quite open to open source, but would be happy to have a go at looking at PR code reviews. I've spent a fair amount of time looking through PRs - if there are more explicit guidelines for code reviewing I would definitely be interested in digesting those. Thanks to @addyosmani for taking up lead of such a great project - it has been instrumental in my journey into programming, javascript and now open source!

**Comment by addyosmani** [comment]
@whitefire0 Thanks! We've been fortunate to have a lot of help from @FadySamirSadek on reviewing PRs lately. 

Would you like to start there (helping with code reviews or issues) and we can add you to the repo too if you feel comfortable with it after a bit?

**Comment by Ahmed-90** [comment]
If you still have a spot available, I would love to help with what I can, especially issues and a little PRs reviewing.

**Comment by FadySamirSadek** [comment]
@Ahmed-90 @whitefire0  Thank you for wanting to contribute to TodoMVC you can start by doing things like reviewing current open PRs and issue. Also, It will be great if you are able to finish some of the tasks we have for the next release on #1880. I am here if you need any help implementing any of these as I just finished my finals yesterday and now I have a lot more time to work on this project and with you. We can also chat on [this project's gitter channel](https://gitter.im/tastejs/todomvc). Sorry if I took a long time to respond.

**Comment by debasree888** [comment]
If you still have one spot available, I would really love to help with what I can, especially issues and PRs reviewing.because by doing this i would get to know a lot of things

**Comment by whitefire0** [comment]
@addyosmani Absolutely; count me in! Feel free to add me to the repo - I will get on the Gitter channel when I get back this evening. @FadySamirSadek no problem on the delay!

**Comment by sr6033** [comment]
I am all up for this. Will love to review the PRs, and help in closing or merging them and in reviewing different issues. I have worked with Javascript and its frameworks all up till now and it would be great helping you out in maintenance.

**Comment by whitefire0** [comment]
Apologies for the delay @fadysamirsadek - will get on these issues as soon as possible.

**Comment by addyosmani** [comment]
Thanks so much for this contribution. I'm going through old issues and closing them out as we last updated the project in a major way in 2024 and want to ensure we're focused on only those issues still relevant. Thanks once again, feel free to reopen if still relevant
